src/sequence_batch_scheduler.h

```diff
@@ -127,6 +137,8 @@ class SequenceBatchScheduler : public Scheduler {
   // The max_sequence_idle_microseconds value for this scheduler.
   uint64_t max_sequence_idle_microseconds_;
 
+  bool stop_;
```

Comment:
Shouldn't this stop_ be a part of scheduler base class? Also, all scheduler seems to be setting this flag to true. So, shouldn't there be a single implementation of Stop function in scheduler base class?

Reply:
I have thought about it, the Scheduler base class was designed to be just an interface so I keep it this way: pure virtual can make the implementer to think twice about what the scheduler should behave in those cases. And now the `Ensemble::Stop()` will just be no-op as discussed.

Reply:
Yes. This can be resolved.